src/Controller/NewsController.php

```diff
@@ -32,20 +37,55 @@ public function show(?News $news): Response
 
 
     #[Route('/admin/news', name: 'admin_news')]
-    public function adminNews(NewsRepository $newsRepository): Response
+    public function adminNews(NewsRepository $newsRepository)
     {
         return $this->render('news/admin/index.html.twig', [
             'news' => $newsRepository->findBy(array(), array('id' => 'DESC'), 15)
         ]);
     }
 
     #[Route('/admin/create', name: 'admin_news_created')]
-    public function createNews(): Response
+    public function createNews(Request $request, SluggerInterface $slugger, Security $security, EntityManagerInterface $entityManager, WebhookDiscordService $webhookDiscordService)
     {
+        $this->webhookDiscordService = $webhookDiscordService;
```

Comment:
Ne set pas ici webhookDiscordService. En fait Symfony te permet de rajouter des services et tes dependances directement en les rajoutant comme arguments dans tes controllers. Tu peux directement utiliser $webhookDiscordService. Si vraiment tu veux vraiment set un service avec this mieux vaut le faire dans un constructeur. Mais tu n'en as pas vraiment besoin ici.